src/modules/site-v2/application.py

```diff
@@ -294,6 +294,10 @@ def _jinja2_filter_species_italic(text):
     text = text.replace('C. elegans', '<i>C. elegans</i>')
     return text
 
+  @app.template_filter('as_percent')

```

Comment:
consider `percent` instead of `as_percent`? Trying align with existing filter names.